src/css/components/visuals/intro.styl

```diff
@@ -95,7 +95,7 @@ $phone_height = 390
   / .splash.is-step-3 &
     opacity 1
     // transition opacity 500ms ease
-    transition opacity, transform
+    transition-property opacity, transform
```

Comment:
You should use `transition` not `transition-property`. Autoprefixer is not able to correctly combine `transition-property` with any other `transition-*` properties: https://github.com/postcss/autoprefixer/blob/master/lib/transition.coffee#L73.